de-lib/src/main/java/org/iplantc/de/server/auth/BaseUrlConnector.java

```diff
@@ -120,7 +123,11 @@ protected String addQueryParam(String uriString, String name, String value) thro
      * @throws IOException if a URI representation is invalid or an encoding error occurs.
      */
     protected String addIpAddress(String uriString, HttpServletRequest request) throws IOException {
-        return addQueryParam(uriString, "ip-address", request.getRemoteAddr());
+        if (uriString.startsWith(terrainBaseUrl)) {
+            return addQueryParam(uriString, "ip-address", request.getRemoteAddr());
```

Comment:
This change was required because the [UAT search endpoint](https://vocabs.ands.org.au/repository/api/lda/aas/the-unified-astronomy-thesaurus/2-0-0/concept.json?_pageSize=50&labelcontains=star) would return a `Bad Request` error if the `ip-address` parameter was included. This should also prevent our users' IP addresses from being sent to any other 3rd-party service endpoints.

Reply:
That makes sense. The only endpoint we really need the IP address for is the bootstrap endpoint anyway. 👍